src/Compilers/CSharp/Portable/Emitter/Model/ArrayTypeSymbolAdapter.cs

```diff
@@ -39,11 +39,25 @@ IEnumerable<int> Cci.IArrayTypeReference.LowerBounds
         {
             get
             {
-                for (int i = 0; i < this.Rank; ++i)
-                    yield return 0;
+                var lowerBounds = this.LowerBounds;
+
+                if (lowerBounds.IsDefault)
```

Comment:
I don't think so. ImmutableArray is not a reference type and is not a Nullable type. Regardless, I prefer code written this way.